javascript/core/src/verbs/util/compare.ts

```diff
@@ -132,3 +139,24 @@ function compareBooleans(
 			throw new Error(`Unsupported boolean comparison operator: [${operator}]`)
 	}
 }
+
+function compareDates(
+	left: Date,
+	right: Date,
+	operator: DateComparisonOperator,
+): 1 | 0 {
+	switch (operator) {
+		case DateComparisonOperator.Equals:
+			return left.toLocaleDateString() === right.toLocaleDateString() ? 1 : 0
+		case DateComparisonOperator.NotEqual:
+			return left.toLocaleDateString() !== right.toLocaleDateString() ? 1 : 0
+		case DateComparisonOperator.After:
+		case DateComparisonOperator.AfterToday:
+			return left > right ? 1 : 0
```

Comment:
I think it would be more clear if you created `right = new Date()` here in the comparison - otherwise, someone using the compareDates function might not know that they need to create their own today date before calling this function.